docker-compose.yml

```diff
@@ -11,16 +11,20 @@ services:
       - db
       - redis
       - yarn
+      - testdb
     environment:
       RAILS_ENV: development
       DATABASE_URL: postgresql://forem:forem@db:5432/PracticalDeveloper_development
+      DATABASE_URL_TEST: postgresql://forem:forem@testdb:5432/Forem_test
       REDIS_SESSIONS_URL: redis://redis:6379
       REDIS_SIDEKIQ_URL: redis://redis:6379
       REDIS_URL: redis://redis:6379
       RACK_TIMEOUT_WAIT_TIMEOUT: 10000
       RACK_TIMEOUT_SERVICE_TIMEOUT: 10000
       STATEMENT_TIMEOUT: 10000
       APP_DOMAIN: localhost:3000
+    tmpfs:
+      - /tmp
     volumes:
       - .:/opt/apps/forem:delegated
     entrypoint: ["dockerize", "-wait", "tcp://db:5432", "-wait", "tcp://redis:6379", "-wait", "file:///opt/apps/forem/vendor/bundle/.bundle_finished", "-timeout", "2700s", "-wait-retry-interval", "10s"]
```

Comment:
I didn't add a `-wait`, `tcp://testdb:5432` here. Perhaps we _should_, but we're waiting for a lot of other things, and running tests is something infrequent enough that blocking startup because of it doesn't feel needed.